### DISCUSSION TITLE
Problem with font in linux snap version for Linux Deepin 20.6

### DISCUSSION THREAD

**Discussion by FYM125** [standardnotes/forum#2587] [General]
Hello.
Notes with text on chinese are appeared as rectangles, like: 口口口口... The text could be copied into another app and there appeared true. It means that need to change font, because most of european fonts cannot output chinese characters. 

Appimage version works well.
Demo on official website works well too.

Best regards and many thanks to this soft!
Alexey

**Comment by atmoio** [comment]
Is this using the plain editor, or another editor?

**Comment by FYM125** [comment]
Sorry, don't understand the question.

Just take Snatdard Notes from snap:
$ sudo snap install standard-notes

And there is a problem with font. If possible to change font in Standard-notes, it could be no such problem.
Then I've take and installed appimage. No problem with font.

And no such problem several month ago with Linux Deepin 20.4. Maybe they changed system default fonts.


***@***.***
 
From: Mo
Date: 2022-06-16 23:59
To: standardnotes/forum
CC: FYM125; Author
Subject: Re: [standardnotes/forum] Problem with font in linux snap version for Linux Deepin 20.6 (Issue #1630)
Is this using the plain editor, or another editor?
